### ISSUE TITLE
Replicate Travis CI/CD with Azure Pipelines

### ISSUE THREAD

**Issue by alysivji** [busy-beaver-dev/busy-beaver#21]
Originally planned on completing #13 with Azure Pipelines so I could [build one to throw away](http://wiki.c2.com/?PlanToThrowOneAway) before starting a work project. Learning CI/CD on top of Azure Pipelines was too much of a reach on my part.

I am familiar with [travis-ci](https://travis-ci.org/) (interface, integrations, secrets management, etc) so I decided to focus on learning CI/CD in this environment.

Still gotta figure out Azure Pipelines with Docker. Docs are fantastic.

#### Notes

- Use the Docker plugin, I have already entered my credentials

#### Resources
- [Azure Pipelines -- Docker Docs](https://docs.microsoft.com/en-us/azure/devops/pipelines/languages/docker?view=vsts&tabs=yaml)
- [Azure Pipelines -- Deploy Docker Webapp](https://docs.microsoft.com/en-us/azure/devops/pipelines/apps/cd/deploy-docker-webapp?view=vsts)
- [Migrate from Travis to Azure Pipelines](https://docs.microsoft.com/en-us/azure/devops/pipelines/migrate/from-travis?view=azure-devops)
- [Azure Pipelines with Python — by example](https://medium.com/@anthonypjshaw/azure-pipelines-with-python-by-example-aa65f4070634)

**Comment by thejessleigh** [comment]
Just curious - is there a particular reason you're moving from Travis to Azure? I've never used Azure Pipelines before, so I'm not sure what the selling points are.

**Comment by alysivji** [comment]
Travis laid off a bunch of developers in Berlin and there are rumors that the product is in zombie mode. I don't like the optics of firing developers and Microsoft has been crushing it for the developer community.

**Comment by thejessleigh** [comment]
Happy to take this on. :)

**Comment by alysivji** [comment]
GitHub actions is probably a better bet. Closing this in favour of #261 261
